### PR TITLE
New package: 'sl-5.04'

### PR DESCRIPTION
**Pull request by kylesusername**

### CODE REVIEW

**Comment by beefcurtains** [comment]
Please update this template to [5.03](https://github.com/beefcurtains/sl) and rebase.

(ok I cheated, sue me :)

**Comment by xtraeme** [comment]
Please follow the commit guidelines explained in CONTRIBUTING.md.

File: srcpkgs/sl/template [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+version="5.02"
+revision=1
+makedepends="ncurses-devel"
+short_desc="Cure your bad habit of mistyping"
+maintainer="kylesusername <kpd@openmailbox.org>"
+license="custom"
+homepage="http://www.github.com/mtoyoda/sl"
+distfiles="https://github.com/mtoyoda/$pkgname/archive/$version.tar.gz"
+checksum="1e5996757f879c81f202a18ad8e982195cf51c41727d3fea4af01fdcbbb5563a"
+only_for_archs="i686 i686-musl x86_64 x86_64-musl"
```
Comment (xtraeme) [inline srcpkgs/sl/template]: Nope, this is not a solution.

File: srcpkgs/sl/template [hunk 2]
```diff
@@ -0,0 +1,29 @@
+#template file for 'sl'
+pkgname="sl"
+version="5.02"
+revision=1
+makedepends="ncurses-devel"
+short_desc="Cure your bad habit of mistyping"
+maintainer="kylesusername <kpd@openmailbox.org>"
+license="custom"
```
Comment (xtraeme) [inline srcpkgs/sl/template]: License=custom is wrong, please be concrete with the license.

**Comment by beefcurtains** [comment]
Rebase to e26c881, mark all the extra commits as fixup and change the commit message.

TYVM!

File: srcpkgs/sl/template [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+maintainer="kylesusername <kpd@openmailbox.org>"
+license="sl"
+homepage="https://github.com/beefcurtains/sl"
+distfiles="https://github.com/beefcurtains/$pkgname/archive/v$version.tar.gz"
+checksum=39ded7253eed8a79d8057e85118443618c9a0dda88fda693eda1e6861bad9a2f
+
+do_install() {
+	vbin sl
+	vbin sl-h
+	ln -s sl SL
```
Comment (xtraeme) [inline srcpkgs/sl/template]: `ln -s sl ${DESTDIR}/usr/bin/SL` instead and remove the vbin below.

**Comment by xtraeme** [comment]
please submit a proper pull request, with a single commit and not 10.

**Comment by beefcurtains** [comment]
https://github.com/edx/edx-platform/wiki/How-to-Rebase-a-Pull-Request

TL;DR: ``git rebase -i <upstream>`` (see ``git log``) change "pick" to "fixup" on every commit except the first.

**Comment by kylesusername** [comment]
Oh no, that went horribly wrong

**Comment by kylesusername** [comment]
Closing to save Travis.

**Comment by beefcurtains** [comment]
If you lost a commit, try ``git reflog``. Everything is still in there (as long as you don't remove your clone).
